script.js

```diff
@@ -0,0 +1,37 @@
+//assign values to numOneEl and numTwoEl and operatorEl
+function defineValues (){
+    let numOneEl = Number(document.getElementById("numOne").value)
+    let operatorEl = document.getElementById("operator")
+    let operatorSel = operatorEl.options[operatorEl.selectedIndex].value
+    let numTwoEl = Number(document.getElementById("numTwo").value)
+    let finalAnswerEl = document.getElementById("finalAns")
+
+    finalAnswerEl.innerHTML = calculate(numOneEl, numTwoEl, operatorSel)
+}
+function calculate(numOneEl, numTwoEl, operatorSel){
+    switch(operatorSel){ 
+        case "add":
+            return (numOneEl) + (numTwoEl)
```

Comment:
On these return statements, numOneEl and numTwoEl do not need to be in parentheses - but if this is a style choice then that's okay! The parentheses will not affect the result in this situation.